io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,11 @@
+host_ip: x.xx.xxx.xx
+peer_ip: y.yy.yyy.yy
+peer_user: "root"
+peer_password: "******"
+host_intf1: "enp1s0f0"
```

Comment:
Please use the format of parameters as other tests.
In this case, it is `host_interfaces` and `peer_interfaces`